src/main/java/com/zepben/cimbend/network/model/NetworkProtoToCim.kt

```diff
@@ -260,7 +260,7 @@ fun toCim(pb: PBConductingEquipment, cim: ConductingEquipment, networkService: N
     }
 
 fun toCim(pb: PBConnectivityNode, networkService: NetworkService): ConnectivityNode =
-    networkService.getOrPutConnectivityNode(pb.mRID()).apply {
```

Comment:
This is rightfully removed as it is a hang over from when you had no direct access to connectivity nodes.

Does the change in behaviour need to be added to the breaking changes? probably not as I doubt it impacts anything, but it means now that if you get the same connectivity node twice it will fail for the second rather than merging the two.

Reply:
I guess there is the ever so _slim_ possibility that it will reveal bugs downstream, so I've added a release note regardless.